v0/destinations/singular/data/SINGULARAndroidSessionConfig.json

```diff
@@ -0,0 +1,145 @@
+[
+  {
+    "destKey": "p",
+    "sourceKeys": "context.os.name",
+    "required": true
+  },
+  {
+    "destKey": "i",
+    "sourceKeys": "context.app.namespace",
+    "required": true
+  },
+  {
+    "destKey": "app_v",
+    "sourceKeys": "context.app.version",
+    "required": true
+  },
+  {
+    "destKey": "ip",
+    "sourceKeys": "request_ip",
+    "required": true
+  },
+  {
+    "destKey": "ve",
+    "sourceKeys": "context.os.version",
+    "required": true
+  },
+  {
+    "destKey": "mo",
+    "sourceKeys": "context.device.model",
+    "required": true
+  },
+  {
+    "destKey": "ma",
+    "sourceKeys": "context.device.manufacturer",
+    "required": true
+  },
+  {
+    "destKey": "lc",
+    "sourceKeys": "context.locale",
+    "required": true
+  },
+  {
+    "destKey": "install_source",
+    "sourceKeys": "properties.referring_application",
+    "required": true,
+    "metadata": {
+      "defaultValue": ""
+    }
+  },
+  {
+    "destKey": "asid",
+    "sourceKeys": "properties.asid",
+    "required": true,
+    "metadata": {
+      "defaultValue": ""
+    }
+  },
+  {
+    "destKey": "openuri",
+    "sourceKeys": "properties.url",
+    "required": true,
+    "metadata": {
+      "defaultValue": ""
+    }
+  },
+  {
+    "destKey": "dnt",
+    "sourceKeys": "context.device.adTrackingEnabled",
+    "required": true
+  },
+  {
+    "destKey": "sessionNotificationName",
+    "sourceKeys": "event",
+    "required": true
+  },
+  {
+    "destKey": "install_ref",
+    "sourceKeys": "properties.install_ref",
+    "required": false
+  },
+  {
+    "destKey": "custom_user_id",
+    "sourceKeys": "userId",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "n",
+    "sourceKeys": "context.app.name",
+    "required": false
+  },
+  {
+    "destKey": "utime",
+    "sourceKeys": "timestamp",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "cn",
+    "sourceKeys": "context.network.carrier",
+    "required": false
+  },
+  {
+    "destKey": "fcm",
+    "sourceKeys": "context.device.token",
+    "required": false
+  },
+  {
+    "destKey": "bd",
+    "sourceKeys": "properties.build",
```

Comment:
We support this as `contextual` object 
```
"app": {
                "version": "1.0",
                "namespace": "com.rudderstack.ios.test.objc",
                "name": "RudderSampleAppObjC",
                "build": "1"
            }
            ```
Source key should be `context.app.build`

Reply:
Done

Reply:
Not seeing this change